README.md

```diff
@@ -6,47 +6,52 @@
 ## What is eZ Platform Enterprise Edition?
 *eZ Platform Enterprise Edition* is commercial CMS (Content Management System) software developed by eZ Systems.
 
-*eZ Platform Enterprise Edition* derives from *[eZ Platform](https://github.com/ezsystems/ezplatform)*. It is composed of a set of bundles. eZ Platform Enterprise Edition, like eZ Platform, is built on top of the Symfony framework (Full Stack). It has been in development since 2014.
+*eZ Platform Enterprise Edition* derives from *[eZ Platform](https://github.com/ezsystems/ezplatform)*.
+It is composed of a set of bundles. eZ Platform Enterprise Edition, like eZ Platform, is built on top of the Symfony framework (Full Stack). It has been in development since 2014.
 
 ### How to get access to eZ Platform Enterprise Edition?
 
-While this meta repository, `ezplatform-ee`, is public to ease installation and upgrades, full access can be obtained in one of three ways:
-- Request an online demo on [ez.no](https://ez.no/Products/eZ-Platform-Enterprise-Edition)
-- As a partner, download trial version from [Partner Portal](http://ez.no/Partner-Portal)
-- As a customer with an eZ Enterprise subscription, get full version from [Service Portal](https://support.ez.no/Downloads).
-  Or by setting up [Composer Authentication Tokens](https://doc.ez.no/display/DEVELOPER/Using+Composer) for use in combination with this repository.
+While this meta repository, `ezplatform-ee`, is public to ease installation and upgrades, the full access can be obtained in one of the following ways:
+
+- Request an online demo on [ez.no](https://ez.no/Products/eZ-Platform-Enterprise-Edition).
+- As a partner, download the trial version from [Partner Portal](http://ez.no/Partner-Portal).
+- As a customer with the eZ Enterprise subscription, get full version from [Service Portal](https://support.ez.no/Downloads).
```

Comment:
Question to @konradoboza @mateuszbieniek @barw4, or @mateuszdebinski - does this link still work? I don't see anything, but that's probably because I'm not a Partner :)

Reply:
It does, requires to be logged in. :)

Reply:
> It does, requires to be logged in. :)

Yeah, I thought so. Too expensive for me. I'll take your word for it ;)